app/src/main/scala/org/alephium/explorer/Application.scala

```diff
@@ -59,7 +58,7 @@ class Application(host: String,
     BlockFlowSyncService(groupNum = groupNum, syncPeriod = syncPeriod, blockFlowClient, blockDao)
 
   val mempoolSyncService: MempoolSyncService =
-    MempoolSyncService(syncPeriod = syncPeriod, blockFlowClient, utransactionDao)
+    MempoolSyncService(syncPeriod = syncPeriod, blockFlowClient, UnconfirmedTxDao)
```

Comment:
let's remove the `UnconfirmedTxDao` trait and use directly the object inside `MempoolSynService` without pasing it as arg no?

Reply:
Yep. This will be removed when `MempoolSyncService` turns into `object MempoolSyncService` so this constructor disappears entirely. Can we wait for this change till then? Each PR will be bare minimum changes required for the targeted class.

Reply:
sure